src/game/enemy/systems.rs

```diff
@@ -123,26 +145,57 @@ pub fn spawn_enemies(
             AnimationTimer(Timer::from_seconds(0.1, TimerMode::Repeating)),
             AnimateSprite,
             Enemy {
-                word: EnemyWord {
-                    word: w.to_string(),
-                    ..default()
-                },
                 ..default()
             },
-            Word(w.len(), w.to_string()), // for testing word uniqueness
+            Word(WordTarget::Enemy(3), w.to_string()), // for testing word uniqueness
         ));
         println!("Spawned enemy: {w}.");
     }
 }
 
-#[allow(dead_code)]
-pub fn print_enemy_words(enemy_q: Query<&Enemy, With<Enemy>>) {
-    if enemy_q.is_empty() {
-        println!("empty enemy_word query");
-    } else {
-        println!("<<--->>");
+/**
+    Catch and handle EnemyShotEvent
+    - Determine enemy event was for
+    - Apply damage to enemy
+    - If enemy is dead, despawn
+    - Else, apply new word, and update health bar
+**/
+#[allow(clippy::too_many_arguments)] //TODO: reduce complexity?
+pub fn catch_shot_event(
+    mut commands: Commands,
+    enemy_word_q: Query<(&Parent, Entity, &Word), With<Word>>,
+    mut enemy_q: Query<(&mut Enemy, &Children)>,
+    mut health_bar_q: Query<(&mut Sprite, &mut Transform), With<HealthBar>>,
+    mut shot_event_reader: EventReader<EnemyShotEvent>,
+    asset_server: Res<AssetServer>,
+    mut word_bank: ResMut<WordBank>,
+    word_q: Query<&Word, (With<Word>, Without<InputText>)>
+) {
+    let font: Handle<Font> = asset_server.load("fonts/fyodor/truetype/Fyodor-BoldCondensed.ttf");
+    for shot in shot_event_reader.iter() {
+        for (parent, entity, word) in &mut enemy_word_q.iter() {
+            if word.0 == WordTarget::Enemy(shot.0) {
+                let mut enemy = enemy_q.get_mut(parent.get()).unwrap();
+                enemy.0.health -= 10;
+                if enemy.0.health == 0 {
+                    commands.entity(parent.get()).despawn_recursive();
+                } else {
+                    commands.entity(entity).despawn_recursive();
+                    commands.entity(parent.get()).with_children(|builder| {
+                        //TODO: update background box to change by word length? how to handle health bar then
+                        spawn_word(builder, word_bank.get_word(WordComplexity::Medium, &word_q).as_str(), &font.clone());
+                    });
+                    //TODO: better way to get the child sprite/transform than this?
+                    for &child in enemy.1 {
```

Comment:
Do we plan to have the health bar fill red instead of just removing the green?

Reply:
I debated on that, went with just the green for now since it *seemed* clear enough, can change it easy enough though if youd think red is better than blank